src/includes/Blocks/AlmaBlock.php

```diff
@@ -152,8 +152,9 @@ public function get_payment_method_data() {
 		$nonce_value = $this->checkout_helper->create_nonce_value( $gateway_id );
 
 		// We get the eligibilites.
-		$eligibilities  = $this->alma_settings->get_cart_eligibilities();
-		$eligible_plans = $this->alma_settings->get_eligible_plans_keys_for_cart( $eligibilities, $gateway_id );
+		$eligibilities  = $this->cart_helper->get_cart_eligibilities();
+		$eligible_plans = $this->cart_helper->get_eligible_plans_keys_for_cart( $eligibilities );
+		$eligible_plans = $this->alma_plan_builder->order_plans( $eligible_plans, $gateway_id );
```

Comment:
The same name for two variables, maybe you can change to "$eligible_plans_orded"

Reply:
Fixed